test/BackendApi.Test/Controllers/ParcelControllerTest.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Security.Claims;
+using Pims.Api.Controllers;
+using Pims.Api.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+using Entity = Pims.Api.Data.Entities;
+using Model = Pims.Api.Models;
+using System.Linq;
+
+namespace PimsApi.Test.Controllers
+{
+    public class ParcelControllerTest
+    {
+        #region Variables
+        private readonly ParcelController _parcelController;
+        private readonly PIMSContext _dbContext;
+        private readonly Entity.Parcel _expectedParcel = new Entity.Parcel()
```

Comment:
Probably should look for a way to include test objects/data through some form of configuration/feed.  Doesn't need to happen in this PR though.

Reply:
I also like seeing helper libraries create to generate objects quickly.  Then the repetitive elements of creating objects is hidden and only the relevant properties are set in the unit test.

Reply:
I agree in principle, but in my mind helper methods provide the same functionality. I would switch to helper libraries when we start having a lot of tests creating the same sorts of objects. Either way the tests (the actual facts) don't include the actual object generation other than the function calls.